### PR TITLE
BioPhysician

### PR DESCRIPTION
**Pull request by Arrow768**
Changes the chemist/pharmacist/virologist into the Pharmacist / Bio-Physician

More Details:
- Renames the Chemistry lab in medical to pharmacy
- Renames the Chemistry access to Pharmacy
- Removes the Virologist alt title from Medical Doctors
- Removes the Medical Doctor Access to Virology
- Changes the name of the (medical) chemist to Pharmacist
- Adds a alt title of Bio-Chemist to Pharmacist
- Gives the Pharmacist (and the Bio-Chemist) access to the Virology Lab

Todo:
- [ ] Check for more references to chemist / virologist and fix them
- [x] Sprites
- [x] Mapping
- [x] Testing

Implements: https://forums.aurorastation.org/topic/11455-reviving-suggestion-merge-virology-and-pharmacist-into-a-new-title-bio-physician/

### IC changelog
`ToDo`

### CODE REVIEW

**Comment by XanderDox** [comment]
Change Biophysician to Biochemist.

The word Biohysician is redundant, an oxymoron and nonexistent. Also, Medical players overwhelming hate the name biophysician.

**Comment by Arrow768** [comment]
!review

File: code/game/jobs/job/medical.dm [hunk 1]
```diff
@@ -130,25 +116,26 @@
 	supervisors = "the chief medical officer"
 	selection_color = "#ffeef0"
 	economic_modifier = 5
-	access = list(access_medical, access_medical_equip, access_morgue, access_surgery, access_chemistry, access_virology, access_genetics)
-	minimal_access = list(access_medical, access_medical_equip, access_chemistry)
-	alt_titles = list("Pharmacist")
-	outfit = /datum/outfit/job/chemist
+	access = list(access_medical, access_medical_equip, access_morgue, access_surgery, access_pharmacy, access_virology, access_genetics)
+	minimal_access = list(access_medical, access_medical_equip, access_pharmacy, access_virology)
+	alt_titles = list("Bio-Chemist")
```
Comment (skull132) [inline code/game/jobs/job/medical.dm]:
```suggestion
	alt_titles = list("Biochemist")
```

No hyphen here, in English.

File: code/modules/events/viral_infection.dm [hunk 2]
```diff
@@ -10,7 +10,7 @@ datum/event/viral_infection/setup()
 	var/list/valid_diseases = list()
 
 	for(var/mob/living/carbon/human/H in mob_list)
-		if(H.mind && H.stat != DEAD && H.is_client_active(5) && H.get_assignment() == "Virologist")
+		if(H.mind && H.stat != DEAD && H.is_client_active(5) && H.get_assignment() == "Bio-Chemist")
```
Comment (skull132) [inline code/modules/events/viral_infection.dm]:
```suggestion
		if(H.mind && H.stat != DEAD && H.is_client_active(5) && H.get_assignment() == "Biochemist")
```

File: code/modules/flufftext/Dreaming.dm [hunk 3]
```diff
@@ -9,7 +9,7 @@ var/list/dreams = list(
 	"the head of personnel","the head of security","a chief engineer","a research director","a chief medical officer",
 	"the detective","the warden","a member of the internal affairs","a station engineer","the janitor","atmospheric technician",
 	"the quartermaster","a cargo technician","the botanist","a shaft miner","the psychologist","the chemist","the geneticist",
-	"the virologist","the roboticist","the chef","the bartender","the chaplain","the librarian","a mouse","an ert member",
+	"the bio-chemist","the roboticist","the chef","the bartender","the chaplain","the librarian","a mouse","an ert member",
```
Comment (skull132) [inline code/modules/flufftext/Dreaming.dm]:
```suggestion
	"the biochemist","the roboticist","the chef","the bartender","the chaplain","the librarian","a mouse","an ert member",
```

File: config/example/tips.txt [hunk 4]
```diff
@@ -21,7 +21,7 @@ As a Medical Doctor, you can attempt to drain blood from a husk with a syringe t
 As a Medical Doctor, you can surgically implant or extract things from people's chests. This can range from putting in a bomb to pulling out an alien larva.
 As a Medical Doctor, you must target the correct limb and be on help intent when trying to perform surgery on someone.
 As a Chemist, there are dozens of chemicals that can heal, and even more that can cause harm. Experiment!
-As the Virologist, you only require small amounts of vaccine to heal a sick patient. Work with the Chemist to distribute your cures more efficiently.
+As the Bio-Chemist, you only require small amounts of vaccine to heal a sick patient. Use your chemistry access to distribute your cures more efficiently.
```
Comment (skull132) [inline config/example/tips.txt]:
```suggestion
As the Biochemist, you only require small amounts of vaccine to heal a sick patient. Use your chemistry access to distribute your cures more efficiently.
```

File: html/changelogs/arrow768-chemist-physician.yml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - tweak: "Removed the Virologist and Chemist. Replaced it with the Pharmacist and Bio-Chemist"
```
Comment (skull132) [inline html/changelogs/arrow768-chemist-physician.yml]:
```suggestion
  - tweak: "Removed the Virologist and Chemist. Replaced it with the Pharmacist and Biochemist"
```

**Comment by fernerr** [comment]
Did another round of testing and it all seems to work. The one thing missing would be making it so Biochemists spawn with virology gear, as voted for in the thread.

**Pull request merged**